lib/edit/a_info.txt

```diff
@@ -4517,3 +4517,70 @@ D:それはいつかなされるだろう，しかし，その前にお前がビ
 D:イェンダーの魔除けを取り戻すことができれば，我々は永遠に安泰だ．
 D:「ダイアナの長弓を持って行くがいい．魔除けを探す冒険の助けとなるだろう．」
 D:(勇者オリオン，JNethack)
+
+N:271:ジャック・オーの
+E:of Jack-o
+I:39:4:1
+W:3:1:10:3000
+P:0:1d1:0:0:0
+F:ACTIVATE
+U:DISP_EVIL
+D:$A fancy lantern in the shape of a pumpkin.
+D:$It has a mysterious power to attract good spirits and keep evil spirits away.
+D:カボチャの形をしたファンシーなランタンだ。
+D:善霊を引き寄せ、悪霊たちを遠ざける不思議な力を持っている。
+
+N:272:『高漸離の筑』
+E:'Ancient harp of Gao Jianli'
+I:19:51:1
+W:60:150:200:60000
+P:0:0d0:0:0:0
+F:WIS | CON
+F:ACTIVATE | TELEPATHY | THROW | RES_FEAR | INSTA_ART
+U:SLEEP
+D:$It's a koto-like instrument, but it's strangely heavy.
+D:$For some reason, lead is charged in the body.
+D:$Of course, it's fun to play.
+D:琴のような楽器だが妙に重い。
+D:なぜか胴に鉛が仕込まれているが、もちろんそれは奏でても楽しい。
+
+N:273:スリザリンの
+E:of Marvolo Gaunt
+I:45:50:-2
+W:60:50:10:100000
+P:0:1d1:0:0:0
+F:CHR
+F:INSTA_ART | CURSED | HEAVY_CURSE | PERMA_CURSE | DRAIN_EXP | 
+F:SEE_INVIS | WARNING | TELEPATHY
+U:ANIM_DEAD
+D:$The middle brother asks for the ability to resurrect the dead,
+D:$ so Death gives him the Resurrection Stone.
+D:二番目の兄は『死』をもっと辱しめてやりたいと思い、
+D:愛する人達を死から呼び戻す力を要求しました。
+D:『死』は川から一個の石を拾い、与えました。
+D:(J・K・ローリング「吟遊詩人ビードルの物語」)
+
+N:274:ダッドリーの
+E:of Dudley
+I:31:1:2
+W:10:10:5:40000
+P:1:0d0:0:0:15
+F:STR | DEX | CHR | FREE_ACT
+U:ID_PLAIN
+D:$"Let's fight like gentlemen." —Dudley
+D:騎士の称号を持つ英国紳士ダッドリーの何時如何なる時も外さないボクシンググローブだ。
+
+N:275:『釘バット』
+I:21:1:0
+E:'Nail Bat'
+W:10:20:20:2800
+P:0:3d4:10:7:0
+F:ACTIVATE | BRAND_POIS
+U:HERO
+D:$It's a terrifying-looking club with messy nails.
+D:$  It seems to be a weapon used in a wargame called "Yakyu" that
+D:$ aims to return to the main team by running through three 
+D:$bastions through the defense of the enemy.
+D:乱雑に釘が打ちつけられた禍々しい外見の棍棒だ。
```

Comment:
For the English description of the Nail Bat, I'd use (could also use "Baseball" in place of "Yakyu" to make the joke more obvious):

D:$It's a terrifying-looking club with messy nails.
D:$  It seems to be a weapon used in a wargame called "Yakyu" where
D:$ the goal of the game is to return to one's team by running through
D:$ three bases guarded by the enemy.